src/Microsoft.Win32.Registry.AccessControl/src/Microsoft.Win32.Registry.AccessControl.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyName>Microsoft.Win32.Registry.AccessControl</AssemblyName>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' == 'netfx'">None</ResourcesSourceOutputDirectory>
-    <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsWindows)' != 'true'">true</GeneratePlatformNotSupportedAssembly>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsWindows)' != 'true'">Access Control List (ACL) APIs are specific to the way in which Windows manages access to resources, and are not supported on this platform.</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
Nit: no comma before "and are"... I'd suggest something a bit tighter as well, e.g.
```
Access Control List (ACL) APIs are part of resource management on Windows and are not supported on this platform.
```